### PR TITLE
add TreeStable Timer

### PR DESCRIPTION
**Pull request by tardyp**

### CODE REVIEW

File: buildbot_travis/ui/src/app/configEditors/projects.tpl.jade [hunk 1]
```diff
@@ -46,6 +46,13 @@ config-page
                         input-tags(tags="project.branches", all-tags="allBranches", placeholder="Add a branch")
                     span.col-sm-4.control-label
                         | use '*' character for matching several branches
+                .form-group
+                    label.col-sm-2.control-label Tree Stable Timer:
+                    .col-sm-4
+                        input.form-control(type="number", ng-model="project.treeStableTimer", name="treeStableTimer"
+                                           placeholder="None")
+                    span.col-sm-6.control-label
+                        | Time in minutes during which Buildbot agregates changes before kicking a build.
```
Comment (seankelly) [inline buildbot_travis/ui/src/app/configEditors/projects.tpl.jade]: It's seconds, isn't it?

Comment (tardyp) [inline buildbot_travis/ui/src/app/configEditors/projects.tpl.jade]: right

File: buildbot_travis/ui/src/app/configEditors/projects.tpl.jade [hunk 2]
```diff
@@ -46,6 +46,13 @@ config-page
                         input-tags(tags="project.branches", all-tags="allBranches", placeholder="Add a branch")
                     span.col-sm-4.control-label
                         | use '*' character for matching several branches
+                .form-group
+                    label.col-sm-2.control-label Tree Stable Timer:
+                    .col-sm-4
+                        input.form-control(type="number", ng-model="project.treeStableTimer", name="treeStableTimer"
+                                           placeholder="None")
+                    span.col-sm-6.control-label
+                        | Time in seconds during which Buildbot agregates changes before kicking a build.
```
Comment (seankelly) [inline buildbot_travis/ui/src/app/configEditors/projects.tpl.jade]: I missed this in my previous comment but there's a typo: aggregates*

I would also use "starting a build".

**Comment by seankelly** [comment]
LGTM.

**Pull request merged**